docs/faq.md

```diff
@@ -0,0 +1,15 @@
+## Why do Builders need unique outputs?
+
+`build_runner` relies on determining a static build graph before starting a
+build - it needs to know every file that may be written and which Builder would
```

Comment:
[nit] Why capitalize Builder here but not below?

Reply:
No idea. going with all capitals for now.